.changeset/fifty-crabs-love.md

```diff
@@ -0,0 +1,22 @@
+---
+"@comet/cms-site": minor
+---
+
+Add `hasRichTextBlockContent` helper
```

Comment:
What do you think about the name?

Reply:
I guess the approval means you think it's fine 😁

Reply:
Yep